src/lib/caches/Twitch.js

```
@@ -0,0 +1,52 @@
+      }
+    }
+  }
+
+  static async populate(logger = Logger('TWITCH')) {
+    return this.#hydrate(logger);
+  }
+
+  static get token() {
+    return this.#cache.getKey('token');
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)